text/0000-general-fund.md

```diff
@@ -88,6 +88,14 @@ Everyone is welcome to weigh in on the merits of a funding request in the forum
 * Be specific, professional and thorough with your proposal.
 * Be respectful and professional in your response to any feedback and criticism on the proposal thread 
 
+# Payout Process
+
+* At the time of payout, an amount of Bitcoin equivalent to the funding amount is paid out to an address provided by the funding requestor only.
```

Comment:
It's Bitcoin or Grin, I think we've said. Not sure this should be specified, as it may change in future.

Reply:
Bump on this comment